test/test_actions.py

```diff
@@ -196,17 +200,19 @@ def test_action_acknowledge(self):
         re = resp['_items']
         self.assertEqual(len(re), 1)
 
-        self.assertEqual(re[0]['host'], rh[0]['_id'])
-        self.assertEqual(re[0]['service'], None)
+        self.assertEqual(re[0]['host'], rh[1]['_id'])
+        self.assertEqual(re[0]['host_name'], rh[1]['name'])
+        self.assertEqual(re[0]['service'], rs[0]['_id'])
+        self.assertEqual(re[0]['service_name'], rs[0]['name'])
         self.assertEqual(re[0]['type'], "ack.add")
         self.assertEqual(re[0]['message'], "User comment")
 
         # -------------------------------------------
         # Delete an acknowledge
         data = {
             "action": "delete",
-            "host": rh[0]['_id'],
-            "service": None,
+            "host": rh[1]['_id'],
+            "service": rs[0]['_id'],
```

Comment:
In this case (and after), you test only on service, but you remove the test in case it's only on host, I think we need test both